internal/driver/driver.go

```diff
@@ -177,7 +177,7 @@ func applyCommandOverrides(cmd string, outputFormat int, v variables) variables
 	}
 
 	if outputFormat == report.Proto || outputFormat == report.Raw {
-		trim, tagfilter, filter = false, false, false
+		trim = false
```

Comment:
Might be good to add a test for the raw command to driver_test.go with a filter making sure it fails before and passes now. A dedicated test for proto command would be good too, though not sure how elegant adding that would be given it's a binary format.

Reply:
I replied to comment above with a way to modify TestParse, which would work nicely for testing that the proto command works as intended. We can confirm filters applied to proto in first invocation of have stuck in the second.

Want to check in before making changes, since I think making these changes will have a reasonable impact on TestParse and likely the names of the testdata files used.